### PR TITLE
 Add `Catch French Beginner 6` wiki article

### PR DESCRIPTION
**Pull request by fajar13k**
## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

## Description

- Wiki article for already ended [Catch French Beginner 6](https://osu.ppy.sh/community/forums/topics/1443016)

### CODE REVIEW

File: wiki/Tournaments/de.md [hunk 1]
```diff
@@ -504,6 +504,7 @@ Nicht offizielle Turniere/Wettkämpfe, die von den Communities veranstaltet werd
 | Catch French Beginner #3 | 15.06.2019 | 20.07.2019 | ::{ flag=FR }:: [MindLight](https://osu.ppy.sh/users/7664694) | ::{ flag=FR }:: [Ladylag](https://osu.ppy.sh/users/2812098) | ::{ flag=FR }:: [Kaeldori](https://osu.ppy.sh/users/962519) |
 | [Catch French Beginner 4](CFB/4) | 25.04.2020 | 30.05.2020 | ::{ flag=CA }:: [Remyria](https://osu.ppy.sh/users/1699875) | ::{ flag=FR }:: [Hecatia15](https://osu.ppy.sh/users/3163012) | ::{ flag=FR }:: [Trifano](https://osu.ppy.sh/users/15126172) |
 | [Catch French Beginner 5](CFB/5) | 16.01.2021 | 21.02.2021 | ::{ flag=FR }:: [Enias](https://osu.ppy.sh/users/8384680) | ::{ flag=FR }:: [redjiii](https://osu.ppy.sh/users/1378728) | ::{ flag=FR }:: [MitsukeZ](https://osu.ppy.sh/users/12943161) |
+| [Catch French Beginner 6](CFB/6) | 13.11.2021 | 19.12.2021 | ::{ flag=FR }:: [JunkoAndHiro](https://osu.ppy.sh/users/14004732) | ::{ flag=FR }:: [iron-wish](https://osu.ppy.sh/users/12264606) | ::{ flag=FR }:: [\_\___________](https://osu.ppy.sh/users/6973354) |
```
Comment (TPGPL) [inline wiki/Tournaments/de.md:507]:
```suggestion
| [Catch French Beginner 6](CFB/6) | 13.11.2021 | 19.12.2021 | ::{ flag=FR }:: [JunkoAndHiro](https://osu.ppy.sh/users/14004732) | ::{ flag=FR }:: [iron-wish](https://osu.ppy.sh/users/12264606) | ::{ flag=FR }:: [\_\_\_\_\_\_\_\_\_\_\_\_](https://osu.ppy.sh/users/6973354) |
```

amazing username for markdown

File: wiki/Tournaments/en.md [hunk 2]
```diff
@@ -504,6 +504,7 @@ Unofficial tournaments/competitions hosted by the communities.
 | Catch French Beginner #3 | 2019-06-15 | 2019-07-20 | ::{ flag=FR }:: [MindLight](https://osu.ppy.sh/users/7664694) | ::{ flag=FR }:: [Ladylag](https://osu.ppy.sh/users/2812098) | ::{ flag=FR }:: [Kaeldori](https://osu.ppy.sh/users/962519) |
 | [Catch French Beginner 4](CFB/4) | 2020-04-25 | 2020-05-30 | ::{ flag=CA }:: [Remyria](https://osu.ppy.sh/users/1699875) | ::{ flag=FR }:: [Hecatia15](https://osu.ppy.sh/users/3163012) | ::{ flag=FR }:: [Trifano](https://osu.ppy.sh/users/15126172) |
 | [Catch French Beginner 5](CFB/5) | 2021-01-16 | 2021-02-21 | ::{ flag=FR }:: [Enias](https://osu.ppy.sh/users/8384680) | ::{ flag=FR }:: [redjiii](https://osu.ppy.sh/users/1378728) | ::{ flag=FR }:: [MitsukeZ](https://osu.ppy.sh/users/12943161) |
+| [Catch French Beginner 6](CFB/6) | 2021-11-13 | 2021-12-19 | ::{ flag=FR }:: [JunkoAndHiro](https://osu.ppy.sh/users/14004732) | ::{ flag=FR }:: [iron-wish](https://osu.ppy.sh/users/12264606) | ::{ flag=FR }:: [\_\___________](https://osu.ppy.sh/users/6973354) |
```
Comment (TPGPL) [inline wiki/Tournaments/en.md:507]:
```suggestion
| [Catch French Beginner 6](CFB/6) | 2021-11-13 | 2021-12-19 | ::{ flag=FR }:: [JunkoAndHiro](https://osu.ppy.sh/users/14004732) | ::{ flag=FR }:: [iron-wish](https://osu.ppy.sh/users/12264606) | ::{ flag=FR }:: [\_\_\_\_\_\_\_\_\_\_\_\_](https://osu.ppy.sh/users/6973354) |
```

File: wiki/Tournaments/CFB/6/en.md [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+- [Livestream](https://www.twitch.tv/catchfrenchlive)
+- [Challonge brackets](https://challonge.com/CFB_6)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1bzOdY-7V3Zarksv__8NGnll3mvTQwoWwXLFugPiOAs0/edit?usp=sharing)**
+
+## Participants
+
+| Seed | Members |
+| :-- | :-- |
+| Top | ::{ flag=BE }:: [Kuzuyo](https://osu.ppy.sh/users/18277357), ::{ flag=FR }:: [RexyChann](https://osu.ppy.sh/users/16965749), ::{ flag=FR }:: [iron-wish](https://osu.ppy.sh/users/12264606), ::{ flag=FR }:: [JunkoAndHiro](https://osu.ppy.sh/users/14004732) |
+| High | ::{ flag=FR }:: [\_\___________](https://osu.ppy.sh/users/6973354), ::{ flag=FR }:: [Issuko](https://osu.ppy.sh/users/11447466), ::{ flag=FR }:: [Mimir](https://osu.ppy.sh/users/7382734), ::{ flag=FR }:: [Utopsia](https://osu.ppy.sh/users/11080305) |
```
Comment (TPGPL) [inline wiki/Tournaments/CFB/6/en.md:65]:
```suggestion
| High | ::{ flag=FR }:: [\_\_\_\_\_\_\_\_\_\_\_\_](https://osu.ppy.sh/users/6973354), ::{ flag=FR }:: [Issuko](https://osu.ppy.sh/users/11447466), ::{ flag=FR }:: [Mimir](https://osu.ppy.sh/users/7382734), ::{ flag=FR }:: [Utopsia](https://osu.ppy.sh/users/11080305) |
```

File: wiki/Tournaments/CFB/6/en.md [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+| Placing | Player |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=FR }:: [JunkoAndHiro](https://osu.ppy.sh/users/14004732) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=FR }:: [iron-wish](https://osu.ppy.sh/users/12264606) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=FR }:: [\_\___________](https://osu.ppy.sh/users/6973354) |
```
Comment (TPGPL) [inline wiki/Tournaments/CFB/6/en.md:77]:
```suggestion
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=FR }:: [\_\_\_\_\_\_\_\_\_\_\_\_](https://osu.ppy.sh/users/6973354) |
```

File: wiki/Tournaments/CFB/6/en.md [hunk 5]
```diff
@@ -0,0 +1,348 @@
...
+
+## Match results
+
+### Grand Finals
+
+Saturday, 18 December 2021:
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **iron-wish** ::{ flag=FR }:: | **7** | 5 | ::{ flag=FR }:: \_\___________ | [#1](https://osu.ppy.sh/community/matches/95389445) |
```
Comment (TPGPL) [inline wiki/Tournaments/CFB/6/en.md:232]:
```suggestion
| **iron-wish** ::{ flag=FR }:: | **7** | 5 | ::{ flag=FR }:: \_\_\_\_\_\_\_\_\_\_\_\_ | [#1](https://osu.ppy.sh/community/matches/95389445) |
```

File: wiki/Tournaments/CFB/6/en.md [hunk 6]
```diff
@@ -0,0 +1,348 @@
...
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **JunkoAndHiro** ::{ flag=FR }:: | **6** | 1 | ::{ flag=FR }:: iron-wish | [#1](https://osu.ppy.sh/community/matches/95096511) |
+
+Sunday, 12 December 2021:
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **RexyChann** ::{ flag=FR }:: | **6** | 1 | ::{ flag=FR }:: Utopsia | [#1](https://osu.ppy.sh/community/matches/95149967) |
+| **\_\___________** ::{ flag=FR }:: | **6** | 2 | ::{ flag=FR }:: RexyChann | [#1](https://osu.ppy.sh/community/matches/95152065) |
```
Comment (TPGPL) [inline wiki/Tournaments/CFB/6/en.md:253]:
```suggestion
| **\_\_\_\_\_\_\_\_\_\_\_\_** ::{ flag=FR }:: | **6** | 2 | ::{ flag=FR }:: RexyChann | [#1](https://osu.ppy.sh/community/matches/95152065) |
```

File: wiki/Tournaments/CFB/6/en.md [hunk 7]
```diff
@@ -0,0 +1,348 @@
...
+
+Saturday, 4 December 2021:
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Kuzuyo ::{ flag=BE }:: | -1 | **0** | ::{ flag=FR }:: **JunkoAndHiro** | *win by default* |
+| RexyChann ::{ flag=FR }:: | 1 | **6** | ::{ flag=FR }:: **iron-wish** | [#1](https://osu.ppy.sh/community/matches/94797662) |
+| **Utopsia** ::{ flag=FR }:: | **0** | -1 | ::{ flag=FR }:: Firelex | *win by default* |
+| leeiplo ::{ flag=FR }:: | -1 | **0** | ::{ flag=FR }:: **MRkayle** | *win by default* |
+| Mimir ::{ flag=FR }:: | 2 | **6** | ::{ flag=FR }:: **\_\___________** | [#1](https://osu.ppy.sh/community/matches/94812153) |
```
Comment (TPGPL) [inline wiki/Tournaments/CFB/6/en.md:265]:
```suggestion
| Mimir ::{ flag=FR }:: | 2 | **6** | ::{ flag=FR }:: **\_\_\_\_\_\_\_\_\_\_\_\_** | [#1](https://osu.ppy.sh/community/matches/94812153) |
```

File: wiki/Tournaments/CFB/6/en.md [hunk 8]
```diff
@@ -0,0 +1,348 @@
...
+| **Issuko** ::{ flag=FR }:: | **0** | -1 | ::{ flag=FR }:: Gibralti | *win by default* |
+
+Sunday, 5 December 2021:
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Issuko ::{ flag=FR }:: | 2 | **6** | ::{ flag=FR }:: **\_\___________** | [#1](https://osu.ppy.sh/community/matches/94864020) |
+| MRkayle ::{ flag=FR }:: | 2 | **6** | ::{ flag=FR }:: **Utopsia** | [#1](https://osu.ppy.sh/community/matches/94866270) |
+| Kuzuyo ::{ flag=BE } | -1 | **0** | ::{ flag=FR }:: **\_\___________** | *win by default* |
```
Comment (TPGPL) [inline wiki/Tournaments/CFB/6/en.md:274]:
```suggestion
| Issuko ::{ flag=FR }:: | 2 | **6** | ::{ flag=FR }:: **\_\_\_\_\_\_\_\_\_\_\_\_** | [#1](https://osu.ppy.sh/community/matches/94864020) |
| MRkayle ::{ flag=FR }:: | 2 | **6** | ::{ flag=FR }:: **Utopsia** | [#1](https://osu.ppy.sh/community/matches/94866270) |
| Kuzuyo ::{ flag=BE } | -1 | **0** | ::{ flag=FR }:: **\_\_\_\_\_\_\_\_\_\_\_\_** | *win by default* |
```

File: wiki/Tournaments/CFB/6/en.md [hunk 9]
```diff
@@ -0,0 +1,348 @@
...
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Kuzuyo** ::{ flag=BE }:: | **5** | 2 | ::{ flag=FR }:: Utopsia | [#1](https://osu.ppy.sh/community/matches/94476333) |
+| **RexyChann** ::{ flag=FR }:: | **0** | -1 | ::{ flag=FR }:: Mimir | *win by default* |
+
+Saturday, 27 November 2021:
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| MissFlufly ::{ flag=FR }:: | 0 | **5** | ::{ flag=FR }:: **\_\___________** | [#1](https://osu.ppy.sh/community/matches/94510041) |
```
Comment (TPGPL) [inline wiki/Tournaments/CFB/6/en.md:289]:
```suggestion
| MissFlufly ::{ flag=FR }:: | 0 | **5** | ::{ flag=FR }:: **\_\_\_\_\_\_\_\_\_\_\_\_** | [#1](https://osu.ppy.sh/community/matches/94510041) |
```

File: wiki/Tournaments/CFB/6/en.md [hunk 10]
```diff
@@ -0,0 +1,348 @@
...
+| **JunkoAndHiro** ::{ flag=FR }:: | **4** | 0 | ::{ flag=FR }:: MissFlufly | [#1](https://osu.ppy.sh/community/matches/94201935) |
+
+Sunday, 21 November 2021:
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Utopsia** ::{ flag=FR }:: | **4** | 2 | ::{ flag=FR }:: Gibralti | [#1](https://osu.ppy.sh/community/matches/94252151) |
+| **Issuko** ::{ flag=FR }:: | **0** | -1 | ::{ flag=FR }:: Firelex | *win by default* |
+| **Mimir** ::{ flag=FR }:: | **0** | -1 | ::{ flag=FR }:: MRkayle | [#1](https://osu.ppy.sh/community/matches/94261867) |
+| \_\___________ ::{ flag=FR }:: | -1 | **0** | ::{ flag=FR }:: **leeiplo** | *win by default* |
```
Comment (TPGPL) [inline wiki/Tournaments/CFB/6/en.md:313]:
```suggestion
| \_\_\_\_\_\_\_\_\_\_\_\_ ::{ flag=FR }:: | -1 | **0** | ::{ flag=FR }:: **leeiplo** | *win by default* |
```

**Review by TPGPL — CHANGES_REQUESTED** [review]
apply the changes from `en` to `fr` as well

**Comment by fajar13k** [comment]
Requesting help from french friends (if you guys are free whenever) 🙏 @Soetch @Ganondorf-1

**Comment by Soetch** [comment]
> Requesting help from french friends (if you guys are free whenever) 🙏 @Soetch @Ganondorf-1

Looking into this :)

**Review by Soetch — DISMISSED** [review]
All good !

- [x]  (translations only) The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

**Review by TPGPL — APPROVED** [review]

**Pull request merged**